src/azure-cli/azure/cli/command_modules/appservice/custom.py

```diff
@@ -3274,13 +3274,13 @@ def _start_ssh_session(hostname, port, username, password):
 
 
 def ssh_webapp(cmd, resource_group_name, name, port=None, slot=None, timeout=None):  # pylint: disable=too-many-statements
-    config = get_site_configs(cmd, resource_group_name, name, slot)
-    if config.remote_debugging_enabled:
-        raise CLIError('remote debugging is enabled, please disable')
-
     import platform
     if platform.system() == "Windows":
         raise CLIError('webapp ssh is only supported on linux and mac')
```

Comment:
Can we make Windows also support this feature? Windows also has ssh command. Do you know the reason?

Reply:
I don't know. I am only trying to unblock azdev. @panchagnula @Juliehzl , kindly comment on this.